main.py

```diff
@@ -19,9 +19,16 @@
 import os
 
 from flask import Flask, request
+from config import configs
 
+from dotenv import load_dotenv
+
+
+load_dotenv()
 
 app = Flask(__name__)
+evn = os.environ.get('FLASK_APP_ENV', 'default')
```

Comment:
Should this be env, short for environment (flip the v and the n)?

Reply:
I saw this being used in stackoverflow so I thought maybe it was a convention but I'll change it to env (could've been a typo perhaps).